### DISCUSSION TITLE
How do I suggest presets for add-ons?

### DISCUSSION THREAD

**Discussion by Samq64** [ScratchAddons/ScratchAddons#3047] [Help] [answered]
How does suggesting presets for add-ons work? I would like to suggest TurboWarp dark mode as a preset for the editor dark theme and customizable colours add-on, but I don't know where to suggest it, or if people can even suggest presets.

**Comment by CST1229** [comment]
I presume you could do so using the normal ways of feedback. The [feedback page](https://scratchaddons.com/feedback), discussions and the suggestions channel in the discord server.

**Comment by WorldLanguages** [comment, marked as answer]
At the moment, it's very likely we won't accept new presets unlike we are sure they would get used (surely, a turbowarp dark mode would be accepted, but not "will_wam's block colors")
In the future, we might add a way for users to explore presets made by different Scratchers in a dedicated section of the settings page.

**Reply by Samq64** [reply]
May I make an issue to add TurboWarp dark mode to the presets?

**Reply by Shluffy** [reply]
@Samq64 the worst that will happen is it gets closed

**Reply by TheColaber** [reply]
@Samq64 an issue already exists, search it on the issue searchbar

**Reply by Samq64** [reply]
> @Samq64 an issue already exists, search it on the issue searchbar

OK, thanks! I searched "TurboWarp dark mode" and nothing popped up, so I thought the issue didn't exist.

**Reply by TheColaber** [reply]
I searched Turbowarp darkmode lol

**Comment by Samq64** [comment]
I'll make a PR then.
